.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "framework"]
 	path = framework
-	url = git@github.com:venediktov/vanilla-rtb.git
+	url = https://github.com/venediktov/vanilla-rtb.git
+[submodule "submodules/redisclient"]
+	path = submodules/redisclient
+	url = https://github.com/nekipelov/redisclient
```

Comment:
so it is not a problem with /deps directory , it's just misspelt , if you fix it and rerun it it should build even with /deps directory